packages/core/src/dialects/ibmi/query-generator-typescript.ts

```diff
@@ -217,4 +219,31 @@ export class IBMiQueryGeneratorTypeScript extends AbstractQueryGenerator {
       table.schema ? this.escape(table.schema) : 'CURRENT SCHEMA',
     ]);
   }
+
+  updateQuery(
+    tableOrModel: TableOrModel,
+    attrValueHash: Record<string, unknown>,
+    options?: UpdateQueryOptions,
+  ): QueryWithBindParams {
+    const updateQuery = super.updateQuery(tableOrModel, attrValueHash, { ...options, returning: false });
+
+    if (options?.returning) {
```

Comment:
can you move this logic to a common util used by db2 & ibmi? `getDb2IbmiSelectFromFinalTable` (yeaah that's a long name)

Reply:
Done.